service/core.go

```diff
@@ -126,25 +126,25 @@ func (s *serviceCore) AuthClient() auth.Client {
 }
 
 func (s *serviceCore) Name() string {
```

Comment:
You're right to point this out. In UberFx code, we usually use "service" and not "application". I believe the reason we kept the config keys "application.*" is to keep parity with existing services and avoid introducing new terms. In a lot of our repos, we use applicationID so it will easier to adopt the same terminology for config. It is translated internally to the standardConfig and the key names match the yaml but everywhere else we use service.